app/src/main/java/com/odysee/app/MainActivity.java

```diff
@@ -953,14 +953,7 @@ public void onClick(View view) {
                     public void onClick(View view) {
                         popupWindow.dismiss();
                         hideNotifications();
-
-                        CustomTabColorSchemeParams.Builder ctcspb = new CustomTabColorSchemeParams.Builder();
-                        ctcspb.setToolbarColor(ContextCompat.getColor(MainActivity.this, R.color.colorPrimary));
-                        CustomTabColorSchemeParams ctcsp = ctcspb.build();
-
-                        CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder().setDefaultColorSchemeParams(ctcsp);
-                        CustomTabsIntent intent = builder.build();
-                        intent.launchUrl(MainActivity.this, Uri.parse("https://odysee.com/@OdyseeHelp:b/Community-Guidelines:c"));
+                        openFileUrl("https://odysee.com/@OdyseeHelp:b/Community-Guidelines:c");
```

Comment:
I would prefer the URL to be in a non-translatable string. This would allow a smaller size update of the APK and a faster compiling time if it needs to be changed. This uRL is used on files which can get the context, so there is no need to pass a context instance anywhere, just replacing the string with a context.getResources().getString()

Reply:
What would the size reduction/compile time reduction be with this change?

Reply:
> Wouldn't it be more complicated as a developer to have to find that URL string?
> 

Current IDEs make easy the finding of anything. On IntelliJ, you can quickly jump to the definition.

> Also, isn't using a hardcoded string faster at runtime than searching in the resources?

Yes, it would be faster for the user, but it won’t be noticed. This code will only run when user clicks the button and when its finger just had touched the screen, the code would already be run

Reply:
Thanks for the changes